src/dodal/devices/i10/i10_apple2.py

```diff
@@ -98,10 +99,10 @@ def __init__(
         """
 
         energy_gap_table_path = Path(
-            look_up_table_dir + "IDEnergy2GapCalibrations.csv",
+            os.path.join(look_up_table_dir, "IDEnergy2GapCalibrations.csv"),
         )
         energy_phase_table_path = Path(
-            look_up_table_dir + "IDEnergy2PhaseCalibrations.csv",
+            os.path.join(look_up_table_dir, "IDEnergy2PhaseCalibrations.csv"),
         )
```

Comment:
Should: I believe that `Path` does the join for you so you can do just `energy_gap_table_path = Path(look_up_table_dir, "IDEnergy2GapCalibrations.csv")`